pytest_falcon/plugin.py

```diff
@@ -141,6 +141,8 @@ def fake_request(self, path, **kwargs):
         environ = create_environ(path, **kwargs)
         resp.environ = environ
         body = self.app(environ, resp)
+        if body:
+            body = [_.encode() if not isinstance(_, bytes) else _ for _ in body]
```

Comment:
Usually, imho `_` as variable name means "something we don't use" (eg. x, `_ = sometuple`), so I'd better use a letter (`p` as `part` or `b` as `body` or `s` as `str`).
Or you can even use a normal name, and split on two lines if we hit the 79 wall:

    body = [part.encode() if not isinstance(part, bytes) else part
            for part in body]

Reply:
I can modify for a better variable name, but I'd argue that this is exactly the use case for `_`:  it is essentially a variable we do not care about anywhere outside of the list comprehension.

Bikeshedding a little on the variable name:  `part` is not really any better imo than `_`.

Reply:
> I can modify for a better variable name, but I'd argue that this is exactly the use case for _: it is 
> essentially a variable we do not care about anywhere outside of the list comprehension.

That's not my understanding. For me `_` is used for "throwaway" variables, i.e. variable we don't use at all, which is not the case here :)

> Bikeshedding

:)
Use whatever name you want but `_`, which has a special meaning *(by convention)* :)

Thanks!